File: src/API/Project-1-Web-API/Models/Passenger.cs
using System.ComponentModel.DataAnnotations;

namespace Project_1_Web_API.Models
{
    public class Passenger
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string? Job { get; set; }
        public string Email { get; set; }
        public int Age { get; set; }
    }
}

File: src/src/app/models/passenger.ts
export interface Passenger {
    id: number;
    firstName: string;
    lastName: string;
    job?: string;
    email: string;
    age: number;
}

export interface PassengerDTO{
    firstName: string;
    lastname: string;
    job?: string;
    email: string;
    age: number;
}
